lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -68,8 +68,8 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 to decode video encoded with this element.
                 """,
                 type: :atom,
-                spec: H264.profile_t() | :auto,
-                default: :auto
+                spec: H264.profile_t() | nil,
```

Comment:
Adjust the docs accordingly

Reply:
You mean to change the description of "profile" option as it is in FFmpeg ? ("Force the limits of an H.264 profile. Overrides all settings.")